ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4821,6 +4821,60 @@ kong_apis:
      - name: request-size-limiting
        config.allowed_payload_size: "{{ small_request_size_limit }}"
 
+  - name: "submitDataExhaustRequest"
+    uris: "{{ data_service_prefix }}/v3/job/request/submit"
+    upstream_url: "{{ analytics_api_service_url }}/job/request/submit"
+    strip_uri: true
+    plugins:
+     - name: jwt
+     - name: cors
+     - "{{ statsd_pulgin }}"
+     - name: acl
+       config.whitelist:
+         - 'dataAccess'
```

Comment:
@sowmya-dixit  it should have dataCreate ACL

Reply:
Updated ACL to dataCreate